src/game/components/Image.hpp

```diff
@@ -1,16 +1,23 @@
+/*
+** EPITECH PROJECT, 2022
+** BMGO
+** File description:
+** Image
+*/
+
 #ifndef GAME_COMPONENTS_IMAGE_HPP_
 #define GAME_COMPONENTS_IMAGE_HPP_
 
 #include <filesystem>
 #include "ecs/Component.hpp"
 #include "raylib/images/Image.hpp"
```

Comment:
Does it exist ? it's not raylib/textures/Image.hpp ?

Reply:
raylib/images/Image.hpp for the raylib object
game/components/Image.hpp for the component